comptests/motr/test_motr_di.py

```diff
@@ -129,16 +128,16 @@ def m0cp_corrupt_data_m0cat(self, layout_ids, bsize_list, count_list, offsets):
                         layout, outfile, node, client_num)
                     self.motr_obj.cp_update_cmd(
                         b_size=b_size, count=cnt_u,
-                        object_id=object_id, layout=layout,
-                        infile=infile, node=node, client_num=client_num, offset=offset)
+                        obj=object_id, layout=layout,
+                        file=infile, node=node, client_num=client_num, offset=offset)
                     self.motr_obj.cat_cmd(b_size, cnt_c, object_id, layout, outfile, node,
                                           client_num)
-                    self.motr_obj.md5sum_cmd(infile, outfile, node)
+                    self.motr_obj.md5sum_cmd(infile, outfile, node, flag=True)
                     self.motr_obj.unlink_cmd(object_id, layout, node, client_num)
 
             logger.info("Stop: Verify multiple m0cp/cat operation")
 
-    @pytest.mark.skip(reason="Feature Unavailable")
+    # @pytest.mark.skip(reason="Feature Unavailable")
```

Comment:
Can you either remove the comment or keep it until your TA is complete?

Reply:
done